web/src/admin/admin-settings/AdminSettingsForm.ts

```diff
@@ -156,8 +157,7 @@ export class AdminSettingsForm extends Form<SettingsRequest> {
             <ak-form-element-horizontal label=${msg("Footer links")} name="footerLinks">
                 <ak-codemirror
                     mode=${CodeMirrorMode.YAML}
-                    .parseValue=${false}
-                    value="${first(this._settings?.footerLinks, [])}"
+                    value="${YAML.stringify(first(this._settings?.footerLinks, []))}"
```

Comment:
```suggestion
                    .value="${first(this._settings?.footerLinks, [])}"
```

with that the element will do the parsing itself

Reply:
also on L164 could you change it to say JSON or YAML

Reply:
With JSON and no parsing, it shows up as 
![image](https://github.com/goauthentik/authentik/assets/18313093/6e19566e-ec2d-4aea-ba28-c49a75c7fc85)

Reply:
Same with YAML.

Reply:
I didn't, that might be it

Reply:
Yep, it indeed does the trick. I left it as YAML though, as with JSON it still shows up as `[object Object]`